src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -244,7 +247,17 @@ public void RemoveIssueTracker(IIssueTracker issueTracker)
             lock (issueTrackers)
             {
                 issueTrackers.Remove(issueTracker);
+
+                // The lifetime of an issue tracker is tied to a single document. A tracker is removed when
+                // it is no longer needed i.e. the document has been closed.
```

Comment:
Can we have multiple taggers for the same document? What will happen then? Is this event raised multiple times?

Reply:
1. Yes.
3. No.

See [ReadMe.md](https://github.com/SonarSource/sonarlint-visualstudio/blob/7543d596f851c0d09b910da7996162d6c781531b/src/Integration.Vsix/SonarLintTagger/README.md?plain=1#L4) for more information.